runtime/wasmer.go

```diff
@@ -28,12 +28,14 @@ import (
 	"fmt"
 	"unsafe"
 
+	xxhash "github.com/OneOfOne/xxhash"
```

Comment:
I'm stuck.  golangcibot keeps complaining that File is not goimports-ed.  I think I need to run something like `go get github.com/OneOfOne/xxhash` on the golandCI machine so that it can access these.  How?

Reply:
have you run `go fmt`? looks like it is just a formatting issue

Reply:
go fmt give me this: 
``` 
emack@ed-Oryx-Pro:~/projects/chainsafe/gossamer$ go fmt
can't load package: package github.com/ChainSafe/gossamer: unknown import path "github.com/ChainSafe/gossamer": cannot find module providing package github.com/ChainSafe/gossamer
```

Reply:
ok. yes! it's working now! thanks.